airflow/providers/databricks/hooks/databricks.py

```diff
@@ -156,26 +156,37 @@ def submit_run(self, json: dict) -> int:
         return response["run_id"]
 
     def list_jobs(
-        self, limit: int = 25, offset: int = 0, expand_tasks: bool = False, job_name: str | None = None
+        self, limit: int = 25, offset: int | None = None, page_token: str | None = None, expand_tasks: bool = False, job_name: str | None = None
```

Comment:
please add the new parameter last (to avoid breaking code of users who use positional arguments)

Reply:
Done!